src/ngraph/pattern/matcher.hpp

```diff
@@ -155,7 +155,7 @@ class NGRAPH_API ngraph::pattern::Matcher
     OutputVector& get_matched_values() { return m_matched_list; }
     void reset() {}
     const std::string& get_name() { return m_name; }
-    std::shared_ptr<Node> get_pattern() { return m_pattern_node.as_single_output_node(); }
+    std::shared_ptr<Node> get_pattern() { return m_pattern_node.get_node_shared_ptr(); }
```

Comment:
This will break patterns that work with multiple output nodes and assume they get back a GOE.